crates/engine/src/rocksdb_engine/mod.rs

```diff
@@ -104,6 +111,16 @@ impl RocksEngine {
         Ok(size)
     }
 
+    /// Gets the max write data size.
+    /// Max write data size = 2 * key + value + `cf_handle_size` + 1008
```

Comment:
Why is 1008?

Reply:
It is copied from previous code. The `1008` is an estimate value of the written size.

Reply:
I think a constant is better than a magic number.

Reply:
done